FakebookNotifications/FakebookNotifications.DataAccess/Models/Interfaces/INotificationsDatabaseSettings.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace FakebookNotifications.DataAccess.Models.Interfaces
+{
+    public interface INotificationsDatabaseSettings
+    {
+        //Collections
+        string NotificationsCollection { get; }
+        string UserCollection { get; }
+
+        //DB Settings
+        string ConnectionString { get; }
+        string DatabaseName { get; }
```

Comment:
the non-secret values of this configuration should just go in appsettings.json. less for us to keep track of with user secrets + env vars